### ISSUE TITLE
UI basic-block and address highlighting are not propegated beyond assembly view

### ISSUE THREAD

**Issue by cythrauL** [Vector35/binaryninja-api#1194]
When using the bbl.set_user_highlight or the function.set_user_instr_highlight functions, the assembly view displays the highlight, but it is not propegated to the medium_level_il basic blocks or instruction lines derived from those assembly lines.

Steps to reproduce:
select a basic block in any view (asm, il), type: ```current_basic_block.set_user_highlight(HighlightStandardColor.BlueHighlightColor)```

select an instruction in any view, type:
```
current_function.set_user_instr_highlight(
here,
HighlightStandardColor.RedHighlightColor)
```

Expected:
The llil and mlil views will also highlight bbls or lines derived from their asm counterparts

What actually happens:
Only the asm view is highlighted.

Fix:
Blocks and instructions derived from highlighted asm instructions should inherit their highlighting.

**Comment by cythrauL** [comment]
infact, it looks like highlighting is broken for anything other than the asm view... The following doesn't work either:

```
current_function.medium_level_il.basic_blocks[0].set_user_highlight(HighlightStandardColor.BlueHighlightColor)
```
 
Version 1.1.1344 (Build ID c25b2ebb)

**Comment by rssor** [comment]
Resolved as of build 1.1.1543-dev. It is worth noting that highlights are only stored/loaded in terms of the native block, so all multiple IL blocks derived from one native block will share a highlight.
